.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: mix test (Emacs ${{matrix.emacs_version}} | Elixir ${{matrix.elixir}} | Erlang/OTP ${{matrix.otp}})
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        emacs_version:
+          # Add more lines like this if you want to test on different Emacs versions.
+          - 27.1
+          - 26.3
+          - 26.1
+          - 25.2
+          - 25.1
+        include:
+          - elixir: 1.8.x
+            otp: 21.3.8.18
+          - elixir: 1.9.x
+            otp: 21.3.8.18
+          - elixir: 1.10.x
+            otp: 21.3.8.18
+          - elixir: 1.11.x
+            otp: 21.3.8.18
+          - elixir: 1.11.x
+            otp: 23.1.4
+    env:
+      MIX_ENV: test
+      ERT_RUNNER_ARGS:
```

Comment:
Instead of adding this here, the tests should not try to read this environment variable